src/Tribe/Post_Transient.php

```diff
@@ -9,211 +9,206 @@
-				}
+			$pre = apply_filters( 'tribe_pre_post_meta_transient_' . $transient, $post_id, $transient );
+			if ( false !== $pre ) {
+				return $pre;
 			}
+		}
 
-			if ( $result ) {
-				/**
-				 * Use this to attach an Action to when the Transient is deleted
-				 *
-				 * @since  4.1
-				 *
-				 * @param int    $post_id   Post ID
-				 * @param string $transient The Post Meta Key
-				 */
-				do_action( 'tribe_deleted_post_meta_transient', $transient, $post_id, $transient );
+		if ( $_wp_using_ext_object_cache ) {
+			$value = wp_cache_get( "tribe_{$transient}-{$post_id}", "tribe_post_meta_transient-{$post_id}" );
+		} else {
+			$meta_timeout = '_transient_timeout_' . $transient;
+			$meta         = '_transient_' . $transient;
+			$value        = get_post_meta( $post_id, $meta, false );
+
+			// if there aren't any values, communicate that it did not fetch data from post transient
+			if ( ! is_array( $value ) || 0 === count( $value ) ) {
+				return false;
 			}
 
-			return $result;
+			// grab the first value, because that's all we care about
+			$value = current( $value );
+
+			if ( $value && ! defined( 'WP_INSTALLING' ) ) {
+				if ( get_post_meta( $post_id, $meta_timeout, true ) < time() ) {
+					$this->delete( $post_id, $transient );
+
+					return false;
+				}
+			}
 		}
 
 		/**
-		 * Fetches the Transient Data
+		 * Attach an action after getting the new Transient
 		 *
 		 * @since  4.1
 		 *
-		 * @param int    $post_id    The Post ID, can also be a WP_Post
-		 * @param string $transient  Post Meta to Fetch
-		 *
+		 * @param int    $post_id   Post ID
+		 * @param string $transient The Post Meta Key
 		 */
-		public function get( $post_id, $transient ) {
-			global $_wp_using_ext_object_cache;
-
-			if ( is_numeric( $post_id ) ) {
-				$post_id = (int) $post_id;
-			} else {
-				$post = get_post( $post_id );
-				$post_id = $post->ID;
-			}
-
-			if ( has_filter( 'tribe_pre_post_meta_transient_' . $transient ) ) {
-				/**
-				 * Attach an action before getting the new Transient
-				 *
-				 * @since  4.1
-				 *
-				 * @param int    $post_id   Post ID
-				 * @param string $transient The Post Meta Key
-				 */
-				$pre = apply_filters( 'tribe_pre_post_meta_transient_' . $transient, $post_id, $transient );
-				if ( false !== $pre ) {
-					return $pre;
-				}
-			}
-
-			if ( $_wp_using_ext_object_cache ) {
-				$value = wp_cache_get( "tribe_{$transient}-{$post_id}", "tribe_post_meta_transient-{$post_id}" );
-			} else {
-				$meta_timeout = '_transient_timeout_' . $transient;
-				$meta = '_transient_' . $transient;
-				$value = get_post_meta( $post_id, $meta, false );
-
-				// if there aren't any values, communicate that it did not fetch data from post transient
-				if ( ! is_array( $value ) || 0 === count( $value ) ) {
-					return false;
-				}
-
-				// grab the first value, because that's all we care about
-				$value = current( $value );
+		return
+			has_filter( 'tribe_post_meta_transient_' . $transient )
```

Comment:
no need to break the line between return and `has_filter`

Reply:
makes sense. I blame auto-formatting.